nubis/bin/consul2unixusers.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pwd
+import ConfigParser
+import optparse
+import consul
+
+USERADD="/usr/sbin/useradd"
```

Comment:
Will implement libuser soon. As a suggestion, it would have saved a bunch of time on both our parts if you made one comment saying "use libuser for chown, mkdir, etc" rather than replying to every single instance of where it could be used.